### ISSUE TITLE
Planar Dimensions - Requested via Note #12

### ISSUE THREAD

**Issue by DarkRisings** [DarkRisings/src#3]
Moving discussion on this to here. @trseagro

Imagine if every room in the game had the potential of six more "hidden" 
directions: Shadow, Wild, Fire, Earth, Air, and Water.   

These directions plus a seventh, prime, would have their vnum saved in the 
room information.   

We would then have to develop some room and sector tags to truly have fun 
with this.   

A new skill to sense flags on a room, and a new spell to identify if a room 
will allow a caster to open a path to these planes would have to be 
developed.  Then some sort of skill or spell would be created to "open" the 
door to this other plane.  An alternate approach would be for a "gate" to be 
loaded passing to that room's vnum.   

With the exception of the wild and shadow rooms, these places would be 
horrible places for the average player to go.  But individual rooms or even 
entire areas could be there.  You could then link rooms and such.   

So if you were familiar with the D&D form of shadow step, where you 
basically walk through the plane of shadows turning a five day journey into 
a five hour journey.   

The average player would have no idea what rooms and areas exist in the game 
currently, so there would always be an element of mystery to it.  As far as 
building these things go: we could either duplicate afiles and change the 
vnums using find replace in the text file, or the a builder could hand build 
an area.   

The amount of actual coding to implement this is entirely dependent on how 
much different we want things.  We can "build" something like this now with 
the code we have, but it is clunky.  I have seen some really cool things 
done with code to work this out in other places.

**Comment by trseagro** [comment]
I am thinking the wild and shadow planes would be places where Tyrin and Syrin have had their way with things.  We need better names though.  What they shouldn't be is heaven and hell.  Both places could have entire cities inhabiting the spaces of other locations on our map.

**Comment by trseagro** [comment]
IF I haven't said it yet.  I am not asking nor suggesting we build a duplicate of every room and area into the game as alternate planes.  I am suggesting that we start with a handful of rooms linked into key places as a proof of concept.  Long term a person could expand the concept or link entire new areas off of these elemental pathways.
